bing_translator.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name        = 'bing_translator'
-  s.version     = '4.6.0'
-  s.date        = '2016-04-22'
+  s.version     = '4.7.0'
```

Comment:
It should go to `5.0.0` due to the breaking change in the signature of `initialize`.